nvdaHelper/remote/injection.cpp

```diff
@@ -341,8 +328,7 @@ BOOL outprocInitialized=FALSE;
  * Initializes the out-of-process code for NVDAHelper 
  * @param secureMode 1 specifies that NVDA is running in seucre mode, 0 says not.

```

Comment:
Since `injection_initialize` no longer accepts `secureMode` as an argument this part of the documentation comment should be removed.